src/components/pages/CreateAccount/CreateAccountPage.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { Layout, Form, Input, Icon, Button, Spin } from 'antd';
+import 'antd/dist/antd.css';
+import './CreateAccountPage.scss';
+import { useHistory } from 'react-router-dom';
+import { axiosInstance } from 'src/interceptors/interceptor';
+import { AxiosResponse } from 'axios';
+import { notificationService } from 'src/services/notification.service';
+
+
+const CreateAccountPage = () => {
+
+    const [username, setUsername] = React.useState<string>("");
+    const [password, setPassword] = React.useState<string>("");
+    const [email, setEmail] = React.useState<string>("");
+    const [isSpinning, setIsSpinning] = React.useState<boolean>(false);
+
+    let history = useHistory();
+
+    const submit = async (usernameV: string, passwordV: string, emailV: string) => {
+        setIsSpinning(true);
+        let data = {
```

Comment:
chyba nie zmieniasz już nigdzie tego obiektu, to może zmienić na `const`? + może dodać jakiś interfejs na tą strukturę?